src/models/dataElementsSet.ts

```diff
@@ -18,7 +18,7 @@ import { ValidationError } from "../utils/validations";
     # [... Array of data elements ...]
 */
 
-export const indicatorTypes = ["global", "sub", "custom"] as const;
+export const indicatorTypes = ["global", "sub", "custom", "subreportable"] as const;
```

Comment:
The indicatorType is used in many places to make decisions about a dataElement (for example: de.indicatorType === "someValue" or de.indicatorType != "someOtherValue"). So, have we considered how this new type should be treated by the app? to discuss with @adrianq